news/2022-10-21-project-loved-october-2022.md

```diff
@@ -0,0 +1,374 @@
+
+Sorry we couldn't provide you with the best for this month, just feel free to vote for *The Worst* in the attached thread!
+
+[![](/wiki/shared/news/2022-10-21-project-loved-october-2022/1856758.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Kobaryo - Bookmaker](https://osu.ppy.sh/beatmapsets/1856758#mania) by [\[Crz\]Rachel](https://osu.ppy.sh/users/8369409)\
+140 – 210 BPM, 5:16 | 6.01★, 6.67★, 7.32★, 7.96★, 8.62★\
+*description written by [RandomeLoL](https://osu.ppy.sh/users/7080063)*
+
+For those already aware, yes this is **The** Bookmaker. Finally making its way from [Rachel's Rabifiles 2](https://etternaonline.com/pack/82195) to here for everyone to enjoy! This is undeniably the most ~~in~~famous iteration of this song ever charted. Its legacy status even carried over to Mania without ever having been uploaded by the original creator.
+
+But what makes this chart so special? Well, if you're too afraid to try it out for yourself, a very rough description would be that [Rachel](https://osu.ppy.sh/users/8369409) made a Speedjack chart out of Chordjacks. It's very dense and daunting at first, until you realize that the chords have been cleverly placed to be as *""comfortable""* as possible.
+
+Other than that, the rest is up to you, the player — to experience. Oh, and do be very careful with your wrists if you're not used to this kind of stuff. You'll thank me later.
+
+[![](/wiki/shared/news/2022-10-21-project-loved-october-2022/585451.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Le Dos-on - A](https://osu.ppy.sh/beatmapsets/585451#mania) by [Elekton](https://osu.ppy.sh/users/4990211)\
+205 – 235.75 BPM, 5:59 | 5.50★, 6.17★\
+*description written by [RandomeLoL](https://osu.ppy.sh/users/7080063)*
+
+Been a long while since [Elekton](https://osu.ppy.sh/users/4990211) was featured here. But they're coming in strong with Le Dos-on's *A*, a very well known jumpstream chart featured two years ago in the [**osu!mania 4K World Cup of 2020**](/wiki/Tournaments/MWC/2020_4K).
+
+So, who's ready to tackle this gargantuan 6 minute marathon? If you do, don't just lay back on your chair and play on auto-pilot as this won't be your run of the mill jumpstream chart. Do expect trickier parts here and there, and your occasional bursts here and there. Even your occasional LNs here and there that some people will still manage to forget that releases are a thing! Other than that, all in all this will be a gauntlet for your stamina.
```

Comment:
```suggestion
So, who's ready to tackle this gargantuan 6-minute marathon? If you do, don't just lay back on your chair and play on autopilot as this won't be your run-of-the-mill jumpstream chart. Do expect trickier parts here and there, and your occasional bursts here and there. Even your occasional LNs can be found here and there such that some people will still manage to forget that releases are a thing! Other than that, all in all this will be a gauntlet for your stamina.
```